src/_data/catalog/widgets.json

```diff
@@ -625,12 +865,32 @@
     "categories": [
       "Cupertino (iOS-style widgets)"
     ],
+    "subcategories": [],
+    "link": "https://api.flutter.dev/flutter/cupertino/CupertinoSliverNavigationBar-class.html",
+    "image": "<img alt='' src='/assets/images/docs/catalog-widget-placeholder.png'>"
```

Comment:
Looks like this puts the placeholder on the CupertinoSliverNavigationBar, and the CupertinoSliverNavigationBar image on CupertinoSliverRefreshControl.